/nusim:
  ros__parameters:
    rate: 100.0 
    x0: -0.06
    y0: 0.8
    theta0: 1.57
    input_noise: 0.0001 
    slip_fraction: 0.005
    basic_sensor_variance: 0.000001
    max_range: 0.75
    angle_min: 0.0
    angle_max: 6.26573
    angle_increment: 0.0174533
    time_increment: 0.0005574
    scan_time: 0.2
    range_min: 0.11999999731779099
    range_max: 3.5
    noise_level: 0.005
    samples: 360
    obstacles:
      x: [-0.6, 0.7, 0.5, -0.25, 1.3]
      y: [-0.8, -0.7, 0.9, 1.7, -0.6]
      r: 0.038
    walls:
      x_length: 5.0
      y_length: 5.0